aiden_app/tools/wtj_scraper_tool.py

```diff
@@ -0,0 +1,14 @@
+from .tool import Tool
+from .utils.wtj_scraper import WelcomeToTheJungleScraper
+
+
+class WelcomeToTheJungleScraperTool(Tool):
+    def __init__(self):
+        self.scraper = WelcomeToTheJungleScraper()
+        super().__init__("IndeedScraper")
+        self.add_tool("search_jobs", self.search_jobs_wrapper)
```

Comment:
I don't see how this is gonna work. In the `ToolAggregator`, the last function with the same name (search_jobs) will overwrite the previous one.